toolversions.sh

```diff
@@ -161,6 +161,9 @@ install_docfx() {
 
 # Logs to both stdout and a build timing log, allowing
 # post-processing to see how long each part of the build takes.
+# The console log is in magenta to stand out, and doesn't include the
+# timing part.
 log_build_action() {
-  echo "$(date -u -Iseconds) $1" | tee -a $REPO_ROOT/build_timing_log.txt
+  echo -e "\e[1;35m$1\e[0m"
```

Comment:
Maybe add a comment explaining what this does?

Reply:
The comment a few lines earlier says that the output is in magenta - is that not enough?

Reply:
Ah yes, I didn't notice that :)